script/appsignal

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+cd "$(dirname "$0")/.."
+
+if [[ $DYNO == $APPSIGNAL_DYNO_ENABLED ]]; then
+    bundle exec appsignal install
```

Comment:
are you sure this is something that needs to be done at runtime? I would expect this to be something you run once locally, and the docs read that way to me too: https://docs.appsignal.com/ruby/installation.html